pvlib/modelchain.py

```diff
@@ -431,6 +437,22 @@ def desoto(self):
 
         self.dc = self.system.scale_voltage_current_power(self.dc).fillna(0)
 
+    def cec(self):
```

Comment:
The copy/paste and diff rendering makes this look a little strange, but the net effect is that it seems that this PR currently removes ``return self`` from ``ModelChain.desoto``.

Reply:
Ah.  That's an inadvertent deletion.  Wonder how the tests passed without `return self`.  Maybe something worth looking into.

Reply:
Most of the the ModelChain methods mutate an object attribute, so returning self is not necessary for how we use it within pvlib. The advantage of return self is that it allows people to write their own custom code such as:

```
mc.dc().ac().losses()
```

Say ``mc.dc()`` returned the default ``None`` instead of ``self``. Then python would try to call ``None.ac()`` which obviously would fail.

We should add assertions such as ``assert mc == mc.cec()`` to many of the tests.